### DISCUSSION TITLE
How to delete live dial and multi page dials from a group

### DISCUSSION THREAD

**Discussion by OlleBonkson** [fastaddons/GroupSpeedDial#105] [Q&A]
I made a live dial in curiosity, just to have a look at it. Wasn't satisfied with the result, so I want to delete it now but can not access it.
It's too long time ago I made it, so restoring using backup is no option any longer..
Any ideas of how to get rid of it now ?     Thanks in advance

**Comment by fastaddons** [comment]
Hello,
Since right click to show context menu will be captured by the nested page, you need to right click on the are outside the page (but still on the dial). For example right click dial name. And if you have dials names hidden, then right click the area around the dial where the shadow is.
Both of these options should show you correct context menu.

**Reply by OlleBonkson** [reply]
Thanks again, it worked. I had no borders (space around corners was 0). Adjusted this one and as you described it then worked. You are the best.... and a quick reply as well. Cheers
